src/constants.js

```diff
@@ -25,6 +25,7 @@ export const VIEW_UPLOAD_SUCCESS: 'upload-success' = 'upload-success';
 export const TYPE_FOLDER: 'folder' = 'folder';
 export const TYPE_FILE: 'file' = 'file';
 export const TYPE_WEBLINK: 'web_link' = 'web_link';
+export const TYPE_VERSIONS: 'versions' = 'versions';
```

Comment:
There shouldn't be a type versions, since thats item type, and thats only the above 3.
If you are looking for a cache prefix, then see CACHE_PREFIX_* below